### PR TITLE
7579 Use transaction for adding derived files

### PR DESCRIPTION
**Pull request by APriestman**

### CODE REVIEW

File: Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/Bundle.properties-MERGED [hunk 1]
```diff
@@ -10,6 +10,10 @@ AinStatusNode.status.title=Status
 AinStatusNode.status.unknown=Unknown
 AutoIngestAdminActions.cancelJobAction.title=Cancel Job
 AutoIngestAdminActions.cancelModuleAction.title=Cancel Module
+AutoIngestAdminActions.disableOCR.error=Failed to disable OCR for case "%s".
```
Comment (eugene7646) [inline Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/Bundle.properties-MERGED:13]: @APriestman This file probably doesn't belong in this PR?

Comment (APriestman) [reply Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/Bundle.properties-MERGED:13]: It doesn't, but I think it's just something we haven't gotten around to merging in.

**Review by eugene7646 — APPROVED** [review]
